api/v1alpha1/flinkcluster_types.go

```diff
@@ -151,10 +151,10 @@ type JobManagerSpec struct {
 
 	// TODO: Memory calculation would be change. Let's watch the issue FLINK-13980.
 
-	// Percentage of off-heap memory in containers, as a safety margin to avoid OOM kill.
+	// Percentage of off-heap memory in containers, as a safety margin to avoid OOM kill, default: 25
 	MemoryOffHeapRatio *int32 `json:"memoryOffHeapRatio,omitempty"`
 
-	// Minimum amount of off-heap memory in containers, as a safety margin to avoid OOM kill.
+	// Minimum amount of off-heap memory in containers, as a safety margin to avoid OOM kill, default: 600
```

Comment:
Need to document the unit of memory, MB or Mi, I'm not sure.

Reply:
Following the kubernetes convention - memory capacity type is Quantity - the type is now Quantity. The units of * .heap.size are basically MB, so we defined them in the same units so that we won't get confused.